zio-json/src/test/scala/mongo4cats/zio/json/MongoCollectionSpec.scala

```diff
@@ -0,0 +1,253 @@
+/*
+ * Copyright 2020 Kirill5k
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package mongo4cats.zio.json
+
+import cats.effect.IO
+import cats.effect.unsafe.IORuntime
+import mongo4cats.bson.ObjectId
+import mongo4cats.client.MongoClient
+import mongo4cats.operations.Filter
+import mongo4cats.embedded.EmbeddedMongo
+import org.scalatest.matchers.must.Matchers
+import org.scalatest.wordspec.AsyncWordSpec
+import zio.json._
+
+import java.time.temporal.ChronoField.MILLI_OF_SECOND
+import java.time.temporal.ChronoUnit
+import java.time.{Instant, LocalDate}
+import scala.concurrent.Future
+
+@scala.annotation.nowarn("msg=never used")
+class MongoCollectionSpec extends AsyncWordSpec with Matchers with EmbeddedMongo {
+
+  override val mongoPort: Int = 12348
```

Comment:
Can you try using different port number here (the one that's not used by other specs)?

There are several test failures because embedded mongo wasn't able to start up due to taken port.